code/modules/client/preferences.dm

```diff
@@ -550,6 +551,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			dat += "<b>PDA Style:</b> <a href='?_src_=prefs;task=input;preference=pda_style'>[pda_style]</a><br>"
 			dat += "<br>"
 			dat += "<b>Crew Objectives:</b> <a href='?_src_=prefs;preference=crewobj'>[(crew_objectives) ? "Yes" : "No"]</a><br>"
+			dat += "<b>Go to the Post-Round Arena?:</b> <a href='?_src_=prefs;preference=eorg_arena'>[(eorg_arena) ? "Yes" : "No"]</a><br>"
```

Comment:
```suggestion
			dat += "<b>Go to the post-round arena?:</b> <a href='?_src_=prefs;preference=eorg_arena'>[(eorg_arena) ? "Yes" : "No"]</a><br>"
```

Not sure this is all that clear, but I can't think of anything all that much better. Best I could come up with was:

Opt into end-round grief?